fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsUtil.java

```diff
@@ -132,16 +133,33 @@ public static boolean getBoolean(Map<String, String> properties, String name) th
         }
     }
 
-    /**
-     * Get mapping properties JSONObject.
-     **/
-    public static JSONObject getMappingProps(String sourceIndex, String indexMapping, String mappingType) {
+    public static List<String> getArrayFields(String indexMapping) {
+        JSONObject mappings = getMapping(indexMapping);
+        if (!mappings.containsKey("_meta")) {
+            return new ArrayList<>();
+        }
+        JSONObject meta = (JSONObject) mappings.get("_meta");
+        if (!meta.containsKey("doris")) {
+            return new ArrayList<>();
+        }
+        JSONObject dorisMeta = (JSONObject) meta.get("doris");
```

Comment:
i do not think add the name `doris`to Ealsticsearch index meta  is suitable

Reply:
In the automatic table creation scenario, users are required to create special mappings for the array fields. trino use it `JsonNode metaNode = nullSafeNode(mappings, "_meta");
JsonNode metaProperties = nullSafeNode(metaNode, "trino");`